src/components/CirclePagination/Circle.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { Color, ColorType, breakpoint } from "../../theme/util";
+
+interface CircleProps {
+  className?: string;
+  diameter?: number;
+  color?: ColorType;
+  isActive?: boolean;
+  bgColor?: ColorType | string;
+}
+
+const Dot = styled.div<CircleProps>`
+  width: ${(p) => p.diameter}px;
+  height: ${(p) => p.diameter}px;
+  margin: auto;
+  border-radius: 50%;
+  background-color: ${(p) => p.color};
+`;
+
+const CircleStyled = styled.div<CircleProps>`
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: ${(p) => p.diameter}px;
+  height: ${(p) => p.diameter}px;
+  border-radius: 50%;
+
+  ${breakpoint.lg`
+    background-color: ${(p: CircleProps) => p.bgColor};
+  `}
+
+  ${(p) =>
+    p.isActive &&
+    css`
+      border-width: 1px;
+      border-style: solid;
+      border-color: ${(p: CircleProps) => p.color};
+    `}
+`;
+
+const Circle = ({
+  className,
+  color = Color.Primary,
+  diameter = 26,
+  isActive,
+  bgColor = Color.BackgroundMain,
```

Comment:
We could set the default background color to `transparent` instead here, that way we don't need to pass a `bgColor` prop when the circle background needs to be the same and the parent component. See attached as an example: 
<img width="523" alt="Screenshot 2020-07-22 at 14 16 51" src="https://user-images.githubusercontent.com/2966898/88175511-39470280-cc26-11ea-91d4-aebac19916bb.png">